### PR TITLE
MEP 296.2.0 

### PR DESCRIPTION
**Pull request by cdelabre**
# MEP

version:

## Checklist MEP

- [ ] version number
- [ ] CI/CD tests pass
- [ ] Add message on data review slack channel
- [ ] ⚠️ I will click on **Merge Pull Request** and **NOT** *on Squash and merge*

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/dbt_project.yml [hunk 1]
```diff
@@ -95,4 +95,7 @@ vars:
   re_data:save_test_history: true
   CI_MATERIALIZATION: none
   CI_SEVERITY: none
-  project_name: "{{ 'passculture-data-' ~ target.name ~ '.region-europe-west1' }}"
+  project_name: "{{ 'passculture-data-prod' if target.name == 'prod' else 'passculture-data-ehp' }}"
```
Comment (valoumiaou) [inline orchestration/dags/data_gcp_dbt/dbt_project.yml:98]: inutile: target.project
cf https://docs.getdbt.com/reference/dbt-jinja-functions/target

Comment (cdelabre) [reply orchestration/dags/data_gcp_dbt/dbt_project.yml:98]: Ok, je me note de fixer cela sur une autre PR.

File: orchestration/dags/data_gcp_dbt/models/intermediate/gcp/int_gcp__bigquery_cost.sql [hunk 2]
```diff
@@ -26,7 +26,7 @@ WITH bq_costs AS (
     STRING_AGG(CONCAT(referenced_table_unn.dataset_id, '.', referenced_table_unn.table_id), "," ORDER BY CONCAT(referenced_table_unn.dataset_id, '.', referenced_table_unn.table_id) ) as referenced_tables,
     sum(coalesce(total_bytes_billed, total_bytes_processed)) as total_bytes, 
     count(*) as total_queries
-  FROM `{{ var('project_name') }}`.INFORMATION_SCHEMA.JOBS_BY_PROJECT queries, 
+  FROM `{{ var('project_name') }}.{{ var('region_name') }}`.INFORMATION_SCHEMA.JOBS_BY_PROJECT queries,
```
Comment (valoumiaou) [inline orchestration/dags/data_gcp_dbt/models/intermediate/gcp/int_gcp__bigquery_cost.sql:29]: {{ target.project }}

Comment (cdelabre) [reply orchestration/dags/data_gcp_dbt/models/intermediate/gcp/int_gcp__bigquery_cost.sql:29]: 👍

**Pull request merged**